components/app.js

```diff
@@ -10,7 +11,16 @@ class App {
   }
 
   handleGetGradesSuccess(grades) {
-    this.gradeTable.updateGrades(grades)
+    gradeTable.updateGrades(grades)
+
+    var sum = 0
+
+    for(var i =0; i<grades.length; i++) {
+      sum += grades[i].grade
+    }
+
+    var average = sum / grades.length
+    pageHeader.updateAverage(average)
```

Comment:
Same problem here, use the property rather than the global variable. By using the global variable, your code is not reusable for other iterations of this class, and we want the code to be adaptable to all circumstances.